.github/workflows/build.yml

```diff
@@ -43,6 +43,11 @@ jobs:
           cd build
           make -j2
 
+      - name: Run tests
+        run: |
+          cd build
+          make test
```

Comment:
You can do `make -C build` and drop the `cd`.
Alternatively: `ctest --test-dir build --output-on-failure`